Python/OTP_verifier.py

```diff
@@ -0,0 +1,15 @@
+import random
+from twilio.rest import Client
+
+otp=random.randint(1000,9999)
```

Comment:
The Random Generatic Logic is not that secure and strong, as far as I understand. You can take a look at this [blog](https://realpython.com/python-random/) for better understanding.